package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "stackblitz-starters",
+  "private": true,
+  "scripts": {
+    "format": "prettier --write '**/*'",
+    "format:check": "prettier --check '**/*'"
+  },
+  "devDependencies": {
+    "prettier": "^2.8.8",
+    "prettier-plugin-svelte": "^2.10.1"
```

Comment:
Using Prettier 2.x instead of the recently release 3.0, to stay closer to what StackBlitz actually uses. We can always update later to stay in sync (roughly).